### PR TITLE
Update structural responsibilities in the `NAT` article

### PR DESCRIPTION
**Pull request by Hiviexd**
also forgot to remove airin and zelq last update

### CODE REVIEW

File: wiki/People/Nomination_Assessment_Team/en.md [hunk 1]
```diff
@@ -181,18 +179,19 @@ The following NAT members are exclusively responsible for structural tasks, and
...
 | ::{ flag=SE }:: [Naxess](https://osu.ppy.sh/users/8129817) | [Aiess](https://github.com/Naxesss/Aiess) and [Mapset Verifier](https://github.com/Naxesss/MapsetVerifier) maintenance |
-| ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) | [NAT/BN website](https://bn.mappersguild.com) development |
+| ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) | [BN Management website](https://bn.mappersguild.com) development |
 | ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099) | NAT leadership |
 
 The following NAT members are primarily evaluators, but also contribute to structural tasks:
 
 | User | Task |
 | :-- | :-- |
+| ::{ flag=DE }:: [Capu](https://osu.ppy.sh/users/2474015) | handling osu!taiko [SEV ratings](/wiki/People/Nomination_Assessment_Team/SEV_rating) |
```
Comment (Walavouchey) [inline wiki/People/Nomination_Assessment_Team/en.md:189]:
```suggestion
| ::{ flag=DE }:: [Capu](https://osu.ppy.sh/users/2474015) | Handling osu!taiko [SEV ratings](/wiki/People/Nomination_Assessment_Team/SEV_rating) |
```

File: wiki/People/Nomination_Assessment_Team/en.md [hunk 2]
```diff
@@ -181,18 +179,19 @@ The following NAT members are exclusively responsible for structural tasks, and
...
 | ::{ flag=BR }:: [Dada](https://osu.ppy.sh/users/9119507) | BN Mentorship organisation |
```
Comment (Walavouchey) [inline wiki/People/Nomination_Assessment_Team/en.md:190]:
```suggestion
| ::{ flag=BR }:: [Dada](https://osu.ppy.sh/users/9119507) | BN mentorship organisation |
```

File: wiki/People/Nomination_Assessment_Team/en.md [hunk 3]
```diff
@@ -181,18 +179,19 @@ The following NAT members are exclusively responsible for structural tasks, and
...
 | ::{ flag=DE }:: [FuJu](https://osu.ppy.sh/users/10773882) | Handling user reports and osu! [SEV ratings](/wiki/People/Nomination_Assessment_Team/SEV_rating) |
 | ::{ flag=CN }:: [Garden](https://osu.ppy.sh/users/2849992) | Handling content reports |
 | ::{ flag=NL }:: [Greaper](https://osu.ppy.sh/users/2369776) | [Nomination Planner](https://bnplanner.greaper.net/beatmaps) and [osu!catch MV plugin](https://github.com/Darius-Wattimena/MapsetChecksCatch) development |
-| ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) | NAT Leadership, [NAT/BN website](https://bn.mappersguild.com) development, handling osu!taiko [SEV ratings](/wiki/People/Nomination_Assessment_Team/SEV_rating) |
+| ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) | NAT Leadership, [BN Management website](https://bn.mappersguild.com) and [osu!taiko MV Plugin](https://github.com/Hiviexd/MVTaikoChecks) development |
```
Comment (Walavouchey) [inline wiki/People/Nomination_Assessment_Team/en.md:194]:
```suggestion
| ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) | NAT Leadership, [BN management website](https://bn.mappersguild.com) and [osu!taiko MV plugin](https://github.com/Hiviexd/MVTaikoChecks) development |
```

Comment (Hiviexd) [reply wiki/People/Nomination_Assessment_Team/en.md:194]: "BN Management" is intended to be a proper noun, will fix the `Plugin` one tho

File: wiki/People/Nomination_Assessment_Team/en.md (hunk 2 above)
Comment (Hiviexd) [reply wiki/People/Nomination_Assessment_Team/en.md:190]: intended to be a proper noun, aka the name of the project

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**